galaxy_ng/app/access_control/statements/standalone.py

```diff
@@ -87,7 +87,7 @@
 
     'repositories/ansible/ansible': [
         {
-            "action": "retrieve",
+            "action": "retrieve, my_permissions",
```

Comment:
```suggestion
            "action": ["retrieve", "my_permissions"],
```

Reply:
Thanks, seems my fingers were moving faster than my brain this morning.